reana_workflow_controller/k8s.py

```diff
@@ -137,12 +152,18 @@ def _build_deployment(self, metadata):
             enable_service_links=False,
             automount_service_account_token=False,
         )
+        labels = {
+            "app": self.deployment_name,
+            "reana_workflow_mode": "session",
+            "reana-run-session-workflow-uuid": str(self.workflow_id),
+            "reana-run-session-owner-id": str(self.owner_id),
```

Comment:
```suggestion
            "reana-run-session-owner-uuid": str(self.owner_id),
```
WDYT? To be coherent with `reana-run-session-workflow-uuid`